accounts/models/connections.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from Gp_Backend.s3_storages import MediaStorage
+
+class connection(models.Model):
+    account = models.OneToOneField("accounts.Account", on_delete=models.CASCADE, related_name='connection')
```

Comment:
- let the relationship be `Foreignkey` instead of `OneToOne` as any account would have more than one connection,
- change "related__name" to "connections" instead of "connection".
- always make a related name plural when the relationship is `one-to-many`.